src/components/Card/index.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types'
+
+import './Card.css';
+import Button from '../Button'
+
+export default class Card extends Component {
+    static propTypes = {
+        text: PropTypes.string.isRequired,
+        id: PropTypes.string.isRequired,
+        moveForward: PropTypes.func.isRequired,
+        moveBackward: PropTypes.func.isRequired,
+        delete: PropTypes.func.isRequired,
+    }
+
+    static defaultProps = {
+        text: 'This is a card. It is a good card.',
+    }
+
+    render() {
+        return (
+            <div className='card'>
+                {this.props.text}
+                <Button onClick={this.props.moveBackward} variant='backward' />
```

Comment:
generally, you would wrap the buttons, so that you can move them independently of the card text